src/patternfly/components/MenuToggle/menu-toggle.hbs

```diff
@@ -69,10 +72,18 @@
   {{/if}}
   {{#ifAny menu-toggle--IsPlain @partial-block}}
     {{#if menu-toggle--HasKebab}}
-      <i class="fas fa-ellipsis-v" aria-hidden=true></i>
+      {{#> menu-toggle-toggle-icon}}
+        <svg class="pf-v5-svg" viewBox="0 0 192 512" fill="currentColor" aria-hidden="true" role="img" width="1em" height="1em"><path d="M96 184c39.8 0 72 32.2 72 72s-32.2 72-72 72-72-32.2-72-72 32.2-72 72-72zM24 80c0 39.8 32.2 72 72 72s72-32.2 72-72S135.8 8 96 8 24 40.2 24 80zm0 352c0 39.8 32.2 72 72 72s72-32.2 72-72-32.2-72-72-72-72 32.2-72 72z"></path></svg>
+      {{/menu-toggle-toggle-icon}}
```

Comment:
One other thing in addition to the a11y errors:

```suggestion
      {{#> menu-toggle-icon}}
        <svg class="pf-v5-svg" viewBox="0 0 192 512" fill="currentColor" aria-hidden="true" role="img" width="1em" height="1em"><path d="M96 184c39.8 0 72 32.2 72 72s-32.2 72-72 72-72-32.2-72-72 32.2-72 72-72zM24 80c0 39.8 32.2 72 72 72s72-32.2 72-72S135.8 8 96 8 24 40.2 24 80zm0 352c0 39.8 32.2 72 72 72s72-32.2 72-72-32.2-72-72-72-72 32.2-72 72z"></path></svg>
      {{/menu-toggle-icon}}
```

Plain examples should have toggle-icon as the wrapper instead of toggle-toggle-icon

Reply:
@thatblindgeye Based on  [their definitions](https://patternfly-pr-6349.surge.sh/components/menus/menu-toggle#usage) and implementation consistency, the kebab in the plain example serves as the actionable icon which is the toggle-toggle-icon. Though I agree, ideally a menu-toggle icon is simply and icon and should be abstracted. [Let's proceed with a followup](https://github.com/patternfly/patternfly/issues/6358)